app/controllers/commodities_controller.rb

```diff
@@ -84,7 +84,11 @@ def xi_commodity
   end
 
   def query_params
-    super.merge(filter: { meursing_additional_code_id: meursing_lookup_result.meursing_additional_code_id })
+    query = { filter: {} }
+    query[:filter][:meursing_additional_code_id] = meursing_lookup_result.meursing_additional_code_id
+    query[:filter][:geographical_area_id] = country if country.present?
+
+    super.merge(query)
```

Comment:
We no longer render declarable headings in the headings controller - everything that can be filtered/has measures now is consolidated here